src/main/java/org/apache/mesos/hdfs/Scheduler.java

```diff
@@ -108,53 +279,62 @@ private void launchNode(SchedulerDriver driver, Offer offer,
           .build();
```

Comment:
We aren't using the default command executor, but are doing multiple things inside our executor. Because of this, we wouldn't want to pass it to the commandInfo. This is the way it passes the main command that the task runs.